koalatreehouse23/extract.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+
+import pandas as pd
+
+from utils import CLIENT_ID
+from utils import CLIENT_SECRET
+from utils import REDIRECT_URI
+from utils import PLAYLIST_ID
+
+
+class ExtractSpotifyPlaylist:
+    def __init__(self, scope="playlist-read-collaborative"):
+        self.scope = scope
+        self.sp = spotipy.Spotify(
+            auth_manager=SpotifyOAuth(
+                client_id=self.CLIENT_ID,
+                client_secret=self.CLIENT_SECRET,
+                redirect_uri=self.REDIRECT_URI,
```

Comment:
These should be the input params for the `__init__` instead of hard-coded values. This way you can decouple the class from the external vairables.